### ISSUE TITLE
Not working anymore

### ISSUE THREAD

**Issue by tarnos12** [cooljingle/memrise-all-typing#9]
Hello,  

You are probably aware of it, but most scripts are not working anymore.  
Here is a screenshot of a console:  

https://i.gyazo.com/6a7c3e8869c888affc11823f29f31cdb.png  

Memrise All Typing  
Memrise Wrong Answer Skip  
Memrise Turbo  
Memrise Peek Next  
Memrise CSS Tweaks  

Those are not working.  
Timer seem to be working, tho I am not 100% sure, at least the animation is not playing, but it's still there on the screen.  
I am not sure why memrise is not releasing those tweaks on their own.  
Even if I am not part of leaderboards, I don't mind "cheating" with timers etc.  
Even tho I am coming to this site to learn and not to compete with other people to see who gets more points...

**Comment by cooljingle** [comment]
Are you using Greasemonkey to run scripts? I believe the scripts no longer work using it - you can try running Tampermonkey instead.

**Comment by tarnos12** [comment]
Hey, it seem to be working better.  
I still get some errors.  

I am used to press "Enter" after typing an answer, which gives me an error from memrise, which unfocus input box, so I have to click on it with my mouse.  
Would be nice if we had some workaround for this too.  

This appears when I press enter after an answer(It's possible to press it more than once before new question appears)  

`too much recursion libs-54820ae4f713.js:1:432775`

Other errors:  
`TypeError: MEMRISE.garden.boxes is undefined[Więcej informacji] eval line 2 > eval:19:13`
```
ERROR: Execution of script 'Memrise CSS Tweaks' failed! sheet.addRule is not a function
userscript.html:2:172
```

It really is a pain to use memrise with this pop up box/recursion error.

**Comment by cooljingle** [comment]
trying disabling the wrong answer skip script - that one remains unfixed and I left it low priority as the main purpose of the script was later implemented by memrise themselves (although it does have some use in skipping presentation boxes). 

If it's definitely another script which is the culprit then let me know the specific one and I'll investigate further.

**Comment by tarnos12** [comment]
I still get the same error when I press enter too fast.  
It seems to be memrise error and it's caused by Memrise Turbo script.  

I turned on/off each script and recursion error happens when using Memrise Turbo.  
All other scripts are working somehow.  
Except Memrise CSS Tweaks which throws the error I posted above.

**Comment by cooljingle** [comment]
I've updated the css tweaks scripts to hopefully fix the issue (0.0.12). I haven't been able to reproduce the other error you get - MEMRISE.garden.boxes refers to an object that no longer exists after the recent Memrise update so I would imagine that either you're running an out of date script or there is another script which hasn't been fixed yet which triggers the error.

**Comment by tarnos12** [comment]
Hey, I think that it was caused by "wrong answer skip script" which I removed.  
Only CSS Tweaks remained, but I will take a look at the updated version.  

I disabled all scripts, but Memrise Turbo, then refreshed the page.  
Then you just have to press the arrow button which leads to the next question and at the same time press enter few times.  
This should make a pop up appear.  

Console errors when a pop up appears:  

```
too much recursion
libs-54820ae4f713.js:1:432775
too much recursion[More information]
memrise-bundle-ad9d7f354413.js:1:29717
```

**Comment by cooljingle** [comment]
Is this using my fork of the script or the original one? Version should say 0.1.15.
I've tried to reproduce on firefox but was unable to do so I don't know what the problem is.

**Comment by tarnos12** [comment]
Yes, I am using version 0.1.15 from you.  
Also Firefox 59.0.2 - 64bit

**Comment by cooljingle** [comment]
I managed to reproduce the error, I've fixed it from happening now (0.1.16).  It should be noted that the whole point of that script is that you don't have to press enter when you've answered correctly, although I can see how you could do it accidentally or out of habit.

**Comment by tarnos12** [comment]
Hey, I am aware of that. It's just a habit :) 
Thanks for the fix
